toolkit/tools/pkg/imagecustomizerlib/cosicommon.go

```diff
@@ -0,0 +1,192 @@
+package imagecustomizerlib
+
+import (
+	"archive/tar"
+	"crypto/sha512"
+	_ "embed"
```

Comment:
Is `embed` import needed?

Reply:
oops it should be removed. I'll do it in follow up pr